inc/class-controller.php

```diff
@@ -92,6 +92,52 @@ public function contentItemSubmit( $params ) {
 		echo $page;
 	}
 
+	/**
+	 * Creates a book from an LTI request
+	 *
+	 * @param $action
+	 * @param $params
+	 * @since 1.4.0
+	 */
+	public function createBook( $action, $params ) {
+		$connector = Database::getConnector();
+		$tool      = new Tool( $connector );
```

Comment:
Pls keep only one space between the variable and the `=` sign, this applies to all occurrences commited in this PR.


This applies to all the assignments committed in this PR.

Reply:
done!